config/initializers/session_store.rb

```diff
@@ -1,7 +1 @@
-# MyEtm::Application.config.session_store :active_record_store,
```

Comment:
Why are we not using the AR store? It should be more secure for an IdP. Do you remember this @louispt1?
Using cookie store is not necessary for the last page cookie the apps share.